client/my-sites/plans/jetpack-plans/product-lightbox/included-product-list.tsx

```diff
@@ -1,15 +1,16 @@
-import { TranslateResult, useTranslate } from 'i18n-calypso';
+import { useTranslate } from 'i18n-calypso';
 import getProductIcon from '../product-store/utils/get-product-icon';
 import slugToSelectorProduct from '../slug-to-selector-product';
+import { ProductDescription } from '../types';
```

Comment:
Nitpick 😄  You could `import type` here

Reply:
I was letting my editor deal with imports automatically 
![image](https://github.com/Automattic/wp-calypso/assets/5714212/1558c8b5-bbf9-4119-99b0-85658ba4a4e5)